dtmi/cloudrail/iol/v317/d262144-1.json

```diff
@@ -0,0 +1,19 @@
+{
+  "@id": "dtmi:CloudRail:IOL:V317:D262144;1",
+  "@type": "Interface",
+  "@context": "dtmi:dtdl:context;2",
+  "displayName": "RUxxxU-MxxE-LIU2PN8X2T-H1151/S724",
+  "description": "ultrasonic sensor\r\nmaximum sensing range 40 cm\r\nM18 housing with teach buttons\r\nswitching and analog output",
```

Comment:
Are the `\r\n` intentional? We cannot guarantee those will be properly escaped.

nit. Uppercase `ultrasonic`